recipes/ESC50/classification/hparams/conv2d_classifier.yaml

```diff
@@ -15,7 +15,7 @@ __set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
 # Set up folders for reading from and writing to
 # Dataset must already exist at `audio_data_folder`
 data_folder: !PLACEHOLDER # e.g., /localscratch/UrbanSound8K
-open_rir_folder: <data_folder>/RIRS # Change if needed
+# open_rir_folder: <data_folder>/RIRS # Change if needed
```

Comment:
why are you not directly removing this comment as well? It does not seems to be used in https://github.com/speechbrain/speechbrain/blob/b1576b759e2ffac50ef93108d23d3b2af9bb92e4/recipes/ESC50/classification/train_classifier.py, nop?

Reply:
I was not entirely sure what the intent behind the ESC50 PR was so I tried not to remove things that may be relevant to someone editing/implementing more things later down the line. We _can_ remove them though, I guess.

Reply:
Yes. And if there are any issues we should be suppose to discover them through the use of the recipe testing tool...